config.nix

```diff
@@ -8,6 +8,7 @@ rec {
   # Run time settings
   bitcoinOverFlat = false;
   totalMoneyAmount = 60000000;
+  richPoorDistr = false;
```

Comment:
Because a development evaluation breaks if isn't defined at all.  `false` is the default value, as far as I can tell, so this should be a no-op.